src/test/java/uk/gov/laa/gpfd/integration/ServerSideErrorIT.java

```diff
@@ -37,4 +55,44 @@ void getReportWithIdShouldReturn500WhenCannotConnectToDb() throws Exception {
 
         Assertions.assertEquals(500, response.getStatus());
     }
+
+    @Test
```

Comment:
Thought - if these tests were run in a different order, would this test break? 
So if test below it created a db then this one ran, would it be upset?

Reply:
For that to happen we would need to have set up parallel execution. Whilst the tests may be executed in a different order getCsvWithIdShouldReturn500WhenCannotConnectToDbForReportTable both sets up and drops the tables required for testing. As a result, no matter when getCsvWithIdShouldReturn500WhenCannotConnectToDbForMappingTable runs, there will not be a mapping table to point at

Reply:
oh yeah it drops the tables thats fine :)